tasks/windows-install.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: make the user
+  win_user:
+    description: "Runs BuildKite jobs. See https://buildkite.com/docs/agent/v3/windows."
+    name: "buildkite-agent"
+    # TODO: Single static file with no isolation between runs is unsafe (but that's unlikely to occur)
+    # TODO: remove the password file afterwards
```

Comment:
The file gets created with "0600", so it's not a burning priority to delete to prevent snooping.